ci-operator/step-registry/cucushift/installer/check/nutanix/fields/cucushift-installer-check-nutanix-fields-commands.sh

```diff
@@ -0,0 +1,199 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+check_result=0
+
+function get_arch() {
+  ARCH=$(uname -m | sed -e 's/aarch64/arm64/' -e 's/x86_64/amd64/')
+  echo "${ARCH}"
+}
+
+function run_install() {
+  echo "install-config.yaml"
+  echo "-------------------"
+  grep -v "password\|username\|pullSecret\|auth" <"${dir}"/install-config.yaml
+  set +o errexit
+  openshift-install --dir="${dir}" create cluster 2>&1 | grep --line-buffered -v 'password\|X-Auth-Token\|UserData:' &
+  wait "$!"
+  ret="$?"
+  echo "Installer exit with code $ret"
+}
+
+if [ ! -f /tmp/yq ]; then
+  curl -L "https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_$(get_arch)" \
```

Comment:
I think if use `from: upi-installer` to run the step, we can skip the yq download step.